hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/diskbalancer/policy/DefaultContainerChoosingPolicy.java

```diff
@@ -74,8 +74,8 @@ public ContainerData chooseContainer(OzoneContainer ozoneContainer,
 
     while (itr.hasNext()) {
       ContainerData containerData = itr.next().getContainerData();
-      if (!inProgressContainerIDs.contains(
-          ContainerID.valueOf(containerData.getContainerID())) &&
+      if (containerData.getBytesUsed() != 0 &&
```

Comment:
There were cases where some containers had negative size due to an unknown bug (see [HDDS-10483](https://issues.apache.org/jira/browse/HDDS-10483)). 

It would be better to use `containerData.getBytesUsed() > 0` than `containerData.getBytesUsed() != 0`.

```suggestion
      if (containerData.getBytesUsed() > 0 &&
```